versions/v1.1.0/templates/koordlet.yaml

```diff
@@ -55,6 +55,10 @@ spec:
               add:
                 - SYS_ADMIN
             privileged: true
+          lifecycle:
+            preStop:
```

Comment:
this is for the helm uninstallation scenario of koordlet? how about add as an post-delete hook for helm chart with an option in value(default = false)
https://helm.sh/docs/topics/charts_hooks/